htdocs/core/tpl/extrafields_list_search_param.tpl.php

```diff
@@ -16,6 +16,21 @@
     {
         $crit = $val;
         $tmpkey = preg_replace('/'.$search_options_pattern.'/', '', $key);
+		if (is_array($val) && array_key_exists('start', $val) && array_key_exists('end', $val)) {
+			// date range from list filters is stored as array('start' => <timestamp>, 'end' => <timestamp>)
+			// start date
+			$param .= '&' . $search_options_pattern.$tmpkey.'_startyear='  . intval(dol_print_date($val['start'], '%Y'));
```

Comment:
why the "intval" ?  The dol_print_date should return a string (what we need for the concat)

Reply:
I removed the intval() as it didn't really make sense, as you pointed out.